packages/common/src/ui/components/loading/BouncingGuy/BouncingGuy.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { FC } from 'react';
-import { GlobalStyles, Typography } from '@mui/material';
+import { GlobalStyles } from '@mui/material';
```

Comment:
this made me sad. The use of the translation in here meant that the `Suspense` was always suspended. not an issue previously.. but it is now. after a lot of time getting to the root of it.. just removed the translation and everything is fine 🙄